apps/laboratory/tests/shared/pages/ModalWalletPage.ts

```diff
@@ -12,23 +12,18 @@ export class ModalWalletPage extends ModalPage {
   }
 
   async openSettings() {
-    await this.page.getByTestId('account-button').click()
     await this.page.getByTestId('wui-profile-button').click()
   }
 
   override async switchNetwork(network: string) {
-    await this.openSettings()
     await this.page.getByTestId('account-switch-network-button').click()
     await this.page.getByTestId(`w3m-network-switch-${network}`).click()
-    await this.page.getByTestId('w3m-header-close').click()
     await this.page.waitForTimeout(2000)
   }
 
   async togglePreferredAccountType() {
-    await this.openSettings()
     await this.page.getByTestId('account-toggle-preferred-account-type').click()
-    await this.page.getByTestId('w3m-header-close').click()
-    await this.page.waitForTimeout(2000)
+    await this.page.waitForTimeout(2500)
```

Comment:
Why increased to `2500` instead of `2000`? Does it just work better or?

Reply:
Made it less flickery, but will try reducing to 2000 again and see if it's stable enough

Reply:
If its more stable lets keep it as is. Just wanted to know the reason 👌